### DISCUSSION TITLE
Unable to start Docker container on Mac

### DISCUSSION THREAD

**Discussion by adamjbradley** [nextcloud/all-in-one#4646] [Questions] [answered]
### Steps to reproduce
1. Install Docker
2. Attempt to create a container
docker run --init --sig-proxy=false --name nextcloud-aio-mastercontainer --restart always --publish 80:80 --publish 8081:8080 --publish 8443:8443 -v /var/run/docker.sock:/var/run/docker.sock:ro -e NEXTCLOUD_MOUNT=/Volumes/Untitled -v nextcloud_aio_mastercontainer:/mnt/docker-aio-config nextcloud/all-in-one:latest-arm64
3. Fails

### Expected behavior
Container should start. 
Jenkins Container using docker.sock starts and runs successfully.
![image](https://github.com/nextcloud/all-in-one/assets/1193259/fcd61070-8c7f-4537-8e9b-c8c5dd9cf654)

### Actual behavior
Receive the following error:

_Cannot connect to the docker socket. Cannot proceed.
Did you maybe remove group read permissions for the docker socket? AIO needs them in order to access the docker socket.
If SELinux is enabled on your host, see https://github.com/nextcloud/all-in-one#are-there-known-problems-when-selinux-is-enabled
If you are on TrueNas SCALE, see https://github.com/nextcloud/all-in-one#can-i-run-aio-on-truenas-scale_

### Host OS
macOS 14.3.1 (23D60) M3
Docker 4.26.1 (131620)

#### Nextcloud AIO version
nextcloud/all-in-one:latest-arm64 (as of today)

**Comment by szaimen** [comment, marked as answer]
Hi, have you tried out https://github.com/nextcloud/all-in-one?tab=readme-ov-file#how-to-run-aio-on-macos?

If that doesnt work, I fear it is an upstream bug in docker Desktop...
